File: beecrowd/1/1080/1080.c
#include <stdio.h>

int main(void) {
    int input, index, maior;
    scanf("%d", &input);
    maior = input;
    for (int i = 2; i <= 100; i ++) {
        scanf("%d", &input);
        if (input > maior) {
            maior = input;
            index = i;
        }
    }
    printf("%d\n%d\n", maior, index);
    return 0;
}

File: beecrowd/1/1080/1080.clj
(defn maior [n]
  (let [input (repeatedly n read)
        biggest (apply max input)]
    [
     biggest
     (first (first (filter #(= biggest (second %)) (map-indexed vector input))))
     ]))

(defn main [n]
  (let [aaa (maior n)]
    (printf "%d\n%d\n" (get aaa 0) (inc (get aaa 1)))))

(main 100)
